nixos/modules/security/pam.nix

```diff
@@ -1202,6 +1222,34 @@ in
       };
     };
 
+    security.pam.zfs = {
```

Comment:
There should be an assertion somewhere to detect if the user has indeed ZFS supported on the system, it does not make sense to have those modules loaded without ZFS support.

Reply:
The option is of by default, anyways.

Reply:
a4962a529ed3ba577c80754acc2746b0b8764c81